apps/src/applab/constants.js

```diff
@@ -18,9 +18,12 @@ export const CAPTURE_TICK_COUNT = 300;
 
 export const defaultFontSizeStyle = '14px';
 
-export const themeOptions = ['classic', 'dark'];
+export const DEFAULT_THEME_INDEX = 0;
+export const CLASSIC_THEME_INDEX = 1;
 
-export const themeDisplayNames = ['Classic', 'Dark'];
+export const themeOptions = ['default', 'classic', 'orange', 'citrus'];
```

Comment:
Are we getting rid of dark theme?

Reply:
yes, dark theme was temporary from the @poorvasingal spec (no design input)